pallets/msa/src/lib.rs

```diff
@@ -1125,18 +1113,17 @@ impl<T: Config> Pallet<T> {
 	// pub fn fetch_msa_keys(msa_id: MessageSourceId) -> Vec<KeyInfoResponse<T::AccountId>> {
 	// 	let mut response = Vec::new();
 	// 	for key in Self::get_msa_keys(msa_id) {
-	// 		if let Ok(_info) = Self::try_get_msa_from_public_key(&key) {
+	// 		if let Ok(_info) = Self::ensure_valid_msa_key(&key) {
 	// 			response.push(KeyInfoResponse { key, msa_id });
 	// 		}
 	// 	}
 
 	// 	response
 	// }
 
-	/// Checks that a key is associated to an MSA and has not been revoked.
+	/// Retrieve MSA Id associated with `key` or return `NoKeyExists`
```

Comment:
This is the new pattern for noting errors from a function:
```suggestion
	/// Retrieve MSA Id associated with `key`
	///
	/// # Errors
	/// * [`Error::NoKeyExists`]
	///
```

Reply:
Moved to #613